draft-ietf-taps-interface.md

```diff
@@ -273,6 +273,8 @@ provides:
 
 - Explicit support for transport-specific features to be applied should that
   particular transport be part of a chosen Protocol Stack;
+  
+- Support for using additional information about the networks over which an endpoint can operate (e.g. Provisioning Domain (PvD) information {{RFC7556}}), to inform the selection between alternate network paths;  
```

Comment:
This reads strangely to me as it is not really the API that provides support for this. Not sure anything needs to be added to this section.